allure-bundle/src/main/java/ru/yandex/qatools/allure/AllureMain.java

```diff
@@ -57,7 +57,7 @@ public static void main(String[] args) throws IOException {
      * @throws IOException if any occurs.
      */
     private static void unpackFace(File outputDirectory) throws IOException {
-        ClassLoader loader = AllureMain.class.getClassLoader();
```

Comment:
In that case I'm definitely need to use `AllureMain.class.getClassLoader()`. The resource that I need to unpack is located in `allure-bundle.jar`, and available from ClassLoader that loaded `AllureMain`